xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogStyle.xml

```diff
@@ -260,15 +261,6 @@
 .entry-author-label {
   display: inline;
 }
-/* Pseudo-tools visible in the edit mode */
-div.publish-message {
-  background-image: url("$xwiki.getSkinFile('icons/silk/page_white_world.png')");
-  background-position: 6px center;
-}
-div.hide-message {
-  background-image: url("$xwiki.getSkinFile('icons/silk/lock_open.png')");
-  background-position: 6px center;
-}
```

Comment:
Indeed. In Font Awesome, an icon is not an image, but... a text character. Knowing this, it's now impossible to reference an icon an a CSS file.